pypesto/store/read_from_hdf5.py

```diff
@@ -111,13 +111,24 @@ def read(self, objective: ObjectiveBase = None) -> Problem:
         problem:
             A problem instance with all attributes read in.
         """
+        # raise warning that problem is not to be used but
+        # only stores information
+        logger.warning('WARNING: You are loading a problem.\nThis problem'
```

Comment:
I think a warning is a bit harsh here. Documenting in the docstring should be enough. Potentially warn if no objective is passed.